helpers/deploymentConfig.ts

```diff
@@ -79,6 +79,15 @@ export const ADDRESSES: PreconfiguredAddresses = {
     acm: "0xbf705C00578d43B6147ab4eaE04DBBEd1ccCdc96",
     timelock: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb", // Sepolia Multisig
   },
+  opbnbtestnet: {
+    vBNBAddress: ethers.constants.AddressZero,
+    WBNBAddress: ethers.constants.AddressZero,
+    VAIAddress: ethers.constants.AddressZero,
+    sidRegistryAddress: ethers.constants.AddressZero,
+    feedRegistryAddress: "0x338b3D0E75bc4B3127813A79C8ECBBa96A7DB70a",
+    acm: "0x049f77F7046266d27C3bC96376f53C17Ef09c986",
+    timelock: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf", // opBNB Multisig
```

Comment:
Could we use the addresses from the npm packages?

Reply:
Yes once open PR's get merged I'll use npm packages